src/models/student.py

```diff
@@ -11,6 +12,87 @@ class Student(db.Model):
     name = db.Column(db.String(200))
     created_at = db.Column(db.DateTime, default=datetime.datetime.utcnow)
 
+    @staticmethod
+    def generate_enrolment_key(mobile, from_helpline):
+        '''
+            this function helps to create a a record of a new student in the database were the from_helpine varibale helps to track was the call from helpline or was manually created and send the enrollment key to the mobile number
+
+            if was from_helpline record the incoming calls also
+            
+            USAGE :
+                Student.generate_enrolment_key(mobile, from_helpline)
+
+            params: 
+                mobile : String required
+                from_helpline : Boolean required
+
+        '''
+        student = Student(stage=app.config['STAGES']['EKG'])
```

Comment:
Why is an Enum not used here?

Reply:
Since there was no enum for this. So, I have created a dict in the config file for now